aiplatform/snippets/embeddings.go

```diff
@@ -28,7 +28,8 @@ import (
 )
 
 func embedTexts(
-	apiEndpoint, project, model string, texts []string, task string) ([][]float32, error) {
+	apiEndpoint, project, model string, texts []string,
+	task string, customOutputDimensionality *int) ([][]float32, error) {
```

Comment:
From https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments, we should be using minimal arguments.

blocker: `apiEndpoint` is generally not specified in samples and should be removed. Looking at the code, it seems like maybe the problem is there's a missing method in the library to facilitate text embedding? If so we can make this not-blocker.

Reply:
SG